master/usage/configuration/node.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Configuring a node IP address and subnet
+---
+
+Each Calico node has key routing configuration that may be specified in a number of
+different ways.  In particular, this includes the IPv4 and IPv6 addresses and 
+subnets used for routing.
+
+To configure the IP address and subnet of your node, you have the options of:
+-  Set appropriate flags on the `calicoctl node run` command
```

Comment:
Might make sense to have a quick section at the top that describes at a high level the order in which calico/node looks for its configuration (env variables, stored config, etc)